sos/plugins/kdump.py

```diff
@@ -27,13 +27,71 @@ class RedHatKDump(KDump, RedHatPlugin):
     files = ('/etc/kdump.conf',)
     packages = ('kexec-tools',)
 
+    def kdump_parse_fs(self):
+        device = None
+        for line in self.kdumpconf:
+            if line.startswith(("ext2", "ext3", "ext4", "nfs", "xfs")):
+                try:
+                    device = line.split()[1]
+                except (IndexError):
+                    return False
+        if device:
+            try:
+                fp = open('/etc/fstab', 'r')
+                lines = fp.readlines()
+                fp.close()
+            except IOError:
+                return False
+
+            for line in lines:
+                if line.startswith((device)):
+                    try:
+                        self.fs = line.split()[1].rstrip('/')
+                        return True
+                    except (IndexError):
+                        return False
+        return False
```

Comment:
Since you don't log anything for the differentiated cases there isn't much point in handling the exceptions here and then signalling success/failure with a boolean return (that's generally frowned on in Python - return a value, and use the exception mechanism to signal exceptional events).